configs/fix-timeouts.sh

```
@@ -0,0 +1,16 @@
+BRANCH="fastdg"
+
+# v0.5.8 + cherry-pick https://github.com/sgl-project/sglang/pull/18111
+# Make sure to set SGLANG_JIT_DEEPGEMM_FAST_WARMUP=1
+cd /sgl-workspace/sglang
+git remote remove origin
+git remote add origin https://github.com/trevor-m/sglang.git
+git fetch origin
+git checkout origin/${BRANCH}
+
+# Increase device timeout from 100s -> 1000s
+cd /sgl-workspace/DeepEP
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)